src/resource-graph/azext_resourcegraph/_params.py

```diff
@@ -30,19 +28,16 @@ def load_arguments(self, _):
     with self.argument_context('graph query') as c:
         c.argument('graph_query', options_list=['--graph-query', '--q', '-q'], required=True,
                    completer=get_generic_completion_list(_QUERY_EXAMPLES), help='Resource Graph query to execute.')
-        c.argument('first', options_list=['--first'], required=False, type=int, default=100,
-                   help='The maximum number of objects to return. Accepted range: 1-5000.')
-        c.argument('skip', options_list=['--skip'], required=False, type=int, default=0,
-                   help='Ignores the first N objects and then gets the remaining objects.')
+        c.argument('first', options_list=['--first'], required=False, type=int, default=None,
+                   help='The maximum number of objects to return. Accepted range: 1-1000. Default value is 100.')
```

Comment:
Why the default is None but it's 100 in help message?

Reply:
100 is set be default in the custom.py if skip token is None. This is to know whether 100 was set explicitly by user or user left it empty. Please let me know if there is a better way to do it!

We have this logic where skip and first override skip token if present. Basically if user is passing us skipToken we're decoding the skip token. But if user passes us skip token AND skip or first, we will override the values from the skip token to those passed by user. 
That's why it is important to know when user explicitly wanted to override the top and skip and when the default values are passed which means no override should be done.